ufl/domain.py

```diff
@@ -154,6 +161,9 @@ def ufl_cell(self):
     def is_piecewise_linear_simplex_domain(self):
         return self._ufl_mesh.is_piecewise_linear_simplex_domain()
 
+    def ufl_strip_data(self):
+        return MeshView(self._ufl_mesh, self._topological_dimension, self._ufl_id)
```

Comment:
If you're doing this, surely you need to strip the mesh.

Reply:
Good spot. Thanks.